[2025-06-22 17:42:20,844] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:42:20] "GET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1" 200 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:42:21,144] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:42:21] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:42:26,572] ERROR log_exception() L1414 Exception on /usuario [POST] - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py L1414
Traceback (most recent call last):
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask_openapi3\scaffold.py", line 127, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP teste\API\app.py", line 92, in predict
    personality = int(label_encoder.inverse_transform([y_pred_encoded])[0])
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: invalid literal for int() with base 10: 'Introvert'
[2025-06-22 17:42:26,584] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:42:26] "[35m[1mPOST /usuario HTTP/1.1[0m" 500 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:46,398] INFO _log() L97   [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000 - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:46,399] INFO _log() L97   [33mPress CTRL+C to quit[0m - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:46,406] INFO _log() L97    * Restarting with stat - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:51,086] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:51] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:51,094] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:51] "[32mGET /openapi HTTP/1.1[0m" 308 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:51,106] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:51] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:51,254] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:51] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:51,257] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:51] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:51,259] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:51] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:54,275] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:54] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:54,303] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:54] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:54,304] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:54] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:54,305] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:54] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:54,435] INFO _log() L97   127.0.0.1 - - [22/Jun/2025 17:43:54] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\werkzeug\_internal.py L97  
[2025-06-22 17:43:58,992] ERROR log_exception() L1414 Exception on /usuario [POST] - call_trace=C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py L1414
Traceback (most recent call last):
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask_cors\extension.py", line 176, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
                                                ^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP Sistemas inteligentes\API\env\Lib\site-packages\flask_openapi3\scaffold.py", line 127, in view_func
    response = func(**func_kwargs)
               ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Administrator\Documents\Engenharia de software\Sistemas inteligentes\MVP teste\API\app.py", line 92, in predict
    personality = int(label_encoder.inverse_transform([y_pred_encoded])[0])
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'LabelEncoder' object has no attribute 'inverse_transform'
